content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -257,6 +251,22 @@ search default.svc.cluster.local svc.cluster.local cluster.local
 options ndots:5
 ```
 
+### Feature availability
+
+The availability of Pod DNS Config and DNS Policy "`None`"" is shown as below.
+
+| k8s version | Feature support |
+| :---------: |:-----------:|
+| 1.14 | Stable |
+| 1.10 | Beta (on by default)|
+| 1.9 | Alpha |
+
+To enable this feature in v1.9, the cluster administrator
```

Comment:
Since 1.9 is already out of window, do we need to include text like this?

Reply:
Good point, given we are at 1.14 already, I will remove these text and only mention it went alpha in 1.9.

Reply:
Updated.